crates/evm/traces/src/lib.rs

```diff
@@ -414,7 +414,7 @@ impl From<&CallTraceStep> for StructLog {
             error: step.error.clone(),
             gas: step.gas,
             gas_cost: step.gas_cost,
-            memory: Some(convert_memory(step.memory.data())),
+            memory: Some(convert_memory(step.memory.context_memory())),
```

Comment:
hmm i find this new name a bit confusing

Reply:
same